.github/workflows/ci.yml

```diff
@@ -22,17 +22,19 @@ jobs:
         uses: styfle/cancel-workflow-action@bb6001c4ea612bf59c3abfc4756fbceee4f870c7 # tag=0.10.0
       - name: Checkout repository
         uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b # tag=v3.0.2
+      - name: Setup pnpm
+        uses: pnpm/action-setup@v2
```

Comment:
Since I'm not setting the version in here, the `pnpm/action-setup` will use the one found in `packageManager` property from the `package.json`